packages/core/package.json

```diff
@@ -19,26 +19,32 @@
     "url": "https://github.com/grafana/faro-web-sdk.git",
     "directory": "packages/core"
   },
-  "main": "dist/cjs/index.js",
-  "module": "dist/esm/index.js",
-  "types": "dist/cjs/index.d.ts",
+  "browser": "./dist/bundle/faro-core.iife.js",
```

Comment:
are you sure this is correct? AFAIK this field is for bundlers use, so propably should still be esm/cjs/amd, not iife.